.gitattributes

```diff
@@ -1,3 +1,5 @@
+* text=auto eol=lf
```

Comment:
maybe revise this comment as necessary

```suggestion
# prevents windows runners from failing on lint
* text=auto eol=lf
```

Reply:
Tweaked the comment and added - indeed I was surprised that `make lint` seems to not fail, but there seem to be some other issues related to the setting